### PR TITLE
fixed issue #63

### PR DESCRIPTION
**Pull request by mukesh14149**

### CODE REVIEW

**Comment by mukesh14149** [comment]
Do i need to anything change in this? @niccokunzmann

**Comment by mukesh14149** [comment]
Yes i also thinking to replace javascript function to links!

File: javascript/links.js~ [hunk 1]
```diff
@@ -0,0 +1,27 @@
+function android()
+{
```
Comment (niccokunzmann) [inline javascript/links.js~]: Please do not commit *~ files. Put a line "*~" into .gitginore and use `git rm` to remove the files.

**Comment by niccokunzmann** [comment]
With the style file, now it looks much better in my eyes.

**Comment by niccokunzmann** [comment]
We can merge this and for the links we can use a new pull-request.

**Comment by niccokunzmann** [comment]
I did not see this .-. Usually one would do one pull-request per issue.

**Comment by mukesh14149** [comment]
Actually i resolved this issue in my pull request fixed issue #60

**Comment by niccokunzmann** [comment]
No problem - it is done!

**Pull request closed without merging**